src/parallel-rustc.md

```diff
@@ -92,3 +104,5 @@ are a bit out of date):
 [tracking]: https://github.com/rust-lang/rust/issues/48685
 [monomorphization]:https://rustc-dev-guide.rust-lang.org/backend/monomorph.html
 [parallel-rustdoc]:https://github.com/rust-lang/rust/issues/82741
+[Arc]:https://doc.rust-lang.org/std/sync/struct.Arc.html
+[OwningRef]:https://doc.rust-lang.org/beta/nightly-rustc/rustc_data_structures/owning_ref/index.html
```

Comment:
It's best to link to nightly since that's the latest version of the compiler:

```suggestion
[OwningRef]:https://doc.rust-lang.org/nightly/nightly-rustc/rustc_data_structures/owning_ref/index.html
```